### ISSUE TITLE
[ic-card] updating card text color to the correct spelling fails accessibility

### ISSUE THREAD

**Issue by GCHQ-Developer-530** [mi6/ic-ui-kit#2155]
## Summary of the bug 
Currently in the card css we have the text color set to `color: var(--ic-architechtural-black) !important;` but this is an incorrect spelling and should be `color: var(--ic-architectural-black) !important;`. 

When updating to the correct spelling of the variable name, the card only shows black text and therefore when slotted into hero, doesn't take on the `var(--ic-theme-text)` like it used to.

Tried adding `!important` to the theme text css line in hero but it has no affect.

## 🪜 How to reproduce 
Tell us the steps to reproduce the problem: 
1. Update the css variable to the correct spelling in ic-card.css
2. Run storybook
3. Go on a hero story with a card in the right content slot
4. See the error 

## 🧐 Expected behaviour 
The behaviour should remain as it is now where the text turns white. Updating the variable name to the correct spelling shouldn't impact the current behaviour.

**Comment by ad9242** [comment]
spelling is wrong, but color issue of text may be fixed

**Comment by MI6-255** [comment]
@GCHQ-Developer-741 to look into as part of dark mode
